### ISSUE TITLE
OpenComputers causes crash with IC2

### ISSUE THREAD

**Issue by brisingraerowing** [MightyPirates/OpenComputers#1771]
With OC and IC2 installed, opening the creative mode Search Items tab causes the following crash:

``` text
---- Minecraft Crash Report ----

WARNING: coremods are present:
  NowWithRendering (hopperducts-mc1.8.8-1.4.6.jar)
  TransformerLoader (OpenComputers-MC1.8.9-1.6.0-rc.1+07534fd.jar)
  MalisisSwitchesPlugin (malisisswitches-1.8.9-1.0.1.jar)
  MalisisCorePlugin (malisiscore-1.8.9-2.1.3.jar)
  IC2core (industrialcraft-2-2.3.225-ex18.jar)
  NowWithRendering (redstonepaste-mc1.8.8-1.7.2.jar)
  dsurroundcore (DynamicSurroundings-1.8.9-2.0.5.6.jar)
  offlineskins (OfflineSkins-1.8.9-v3c.jar)
  LoadingPlugin (RandomThings-MC1.8.9-3.6.6.jar)
Contact their authors BEFORE contacting forge

// Oh - I know what I did wrong!

Time: 4/20/16 3:10 PM
Description: Updating screen events

java.lang.NullPointerException: Updating screen events
    at ic2.core.item.ElectricItemTooltipHandler.drawTooltips(ElectricItemTooltipHandler.java:27)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_512_ElectricItemTooltipHandler_drawTooltips_ItemTooltipEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
    at net.minecraftforge.event.ForgeEventFactory.onItemTooltip(ForgeEventFactory.java:202)
    at net.minecraft.item.ItemStack.func_82840_a(ItemStack.java:751)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.updateFilteredItems(GuiContainerCreative.java:367)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_147053_i(GuiContainerCreative.java:345)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_147050_b(GuiContainerCreative.java:518)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_146286_b(GuiContainerCreative.java:427)
    at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:565)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_146274_d(GuiContainerCreative.java:534)
    at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:524)
    at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1674)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1024)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:349)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
    at ic2.core.item.ElectricItemTooltipHandler.drawTooltips(ElectricItemTooltipHandler.java:27)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_512_ElectricItemTooltipHandler_drawTooltips_ItemTooltipEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
    at net.minecraftforge.event.ForgeEventFactory.onItemTooltip(ForgeEventFactory.java:202)
    at net.minecraft.item.ItemStack.func_82840_a(ItemStack.java:751)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.updateFilteredItems(GuiContainerCreative.java:367)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_147053_i(GuiContainerCreative.java:345)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_147050_b(GuiContainerCreative.java:518)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_146286_b(GuiContainerCreative.java:427)
    at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:565)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_146274_d(GuiContainerCreative.java:534)
    at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:524)

-- Affected screen --
Details:
    Screen name: net.minecraft.client.gui.inventory.GuiContainerCreative

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityPlayerSP['BrisingrAerowing'/1488, l='MpServer', x=-354.08, y=69.06, z=-341.45]]
    Chunk stats: MultiplayerChunkCache: 604, 604
    Level seed: 0
    Level generator: ID 00 - default, ver 1. Features enabled: false
    Level generator options: 
    Level spawn location: 120.00,64.00,248.00 - World: (120,64,248), Chunk: (at 8,4,8 in 7,15; contains blocks 112,0,240 to 127,255,255), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
    Level time: 5101 game time, 5101 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
    Forced entities: 70 total; [EntityItem['item.tile.gravel'/16387, l='MpServer', x=-325.91, y=20.00, z=-341.66], EntityItem['item.tile.gravel'/22545, l='MpServer', x=-383.63, y=20.00, z=-366.44], EntityItem['item.tile.sapling.birch'/23605, l='MpServer', x=-356.88, y=65.50, z=-348.88], EntityItem['item.tile.doublePlant.syringa'/23793, l='MpServer', x=-327.63, y=63.00, z=-345.97], EntityItem['item.item.bean'/24831, l='MpServer', x=-379.19, y=63.00, z=-283.97], EntityItem['item.item.bean'/21790, l='MpServer', x=-377.44, y=63.00, z=-285.41], EntityRabbit['Rabbit'/14711, l='MpServer', x=-366.97, y=65.00, z=-337.28], EntityRabbit['Rabbit'/14712, l='MpServer', x=-374.34, y=66.50, z=-329.78], EntityRabbit['Rabbit'/14713, l='MpServer', x=-361.41, y=68.25, z=-326.30], EntityRabbit['Rabbit'/14714, l='MpServer', x=-381.06, y=67.00, z=-333.72], EntityRabbit['Rabbit'/14715, l='MpServer', x=-385.50, y=67.00, z=-325.50], EntityRabbit['Rabbit'/14716, l='MpServer', x=-386.53, y=67.50, z=-323.25], EntityRabbit['Rabbit'/14717, l='MpServer', x=-348.54, y=68.25, z=-329.11], EntityRabbit['Rabbit'/14718, l='MpServer', x=-354.47, y=64.00, z=-325.44], EntitySheep['Sheep'/14719, l='MpServer', x=-318.50, y=64.00, z=-364.50], EntitySheep['Sheep'/14720, l='MpServer', x=-316.50, y=64.00, z=-364.50], EntitySheep['Sheep'/14721, l='MpServer', x=-317.28, y=64.00, z=-363.16], EntityItem['item.tile.gravel'/21889, l='MpServer', x=-330.63, y=30.00, z=-361.41], EntitySheep['Sheep'/14722, l='MpServer', x=-306.84, y=64.50, z=-357.16], EntitySheep['Sheep'/14723, l='MpServer', x=-308.50, y=69.00, z=-321.50], EntitySheep['Sheep'/14724, l='MpServer', x=-309.50, y=63.00, z=-320.50], EntityItem['item.tile.doublePlant.syringa'/24970, l='MpServer', x=-325.31, y=63.00, z=-346.75], EntityItem['item.tile.gravel'/21907, l='MpServer', x=-384.50, y=18.00, z=-366.53], EntityRabbit['Rabbit'/14752, l='MpServer', x=-433.50, y=73.00, z=-358.50], EntityRabbit['Rabbit'/14753, l='MpServer', x=-432.66, y=73.00, z=-358.66], EntityRabbit['Rabbit'/14754, l='MpServer', x=-434.66, y=73.00, z=-359.66], EntitySheep['Sheep'/14756, l='MpServer', x=-307.50, y=63.00, z=-408.50], EntitySheep['Sheep'/14757, l='MpServer', x=-302.28, y=64.50, z=-411.47], EntitySheep['Sheep'/14758, l='MpServer', x=-302.50, y=65.00, z=-413.50], EntityItem['item.tile.gravel'/21928, l='MpServer', x=-387.44, y=20.00, z=-367.41], EntitySheep['Sheep'/14790, l='MpServer', x=-380.50, y=63.00, z=-273.50], EntitySheep['Sheep'/14791, l='MpServer', x=-382.50, y=63.00, z=-273.81], EntitySheep['Sheep'/14792, l='MpServer', x=-382.50, y=63.00, z=-272.19], EntitySheep['Sheep'/14793, l='MpServer', x=-385.16, y=63.00, z=-270.91], EntityPlayerSP['BrisingrAerowing'/1488, l='MpServer', x=-354.08, y=69.06, z=-341.45], EntityItem['item.tile.gravel'/22004, l='MpServer', x=-304.41, y=9.00, z=-345.47], EntityItem['item.tile.gravel'/22005, l='MpServer', x=-302.41, y=10.00, z=-340.56], EntityItem['item.tile.gravel'/22006, l='MpServer', x=-301.41, y=10.00, z=-342.50], EntityBat['Bat'/23115, l='MpServer', x=-429.25, y=31.84, z=-295.56], EntityBat['Bat'/24160, l='MpServer', x=-298.25, y=27.10, z=-293.25], EntitySquid['Squid'/23149, l='MpServer', x=-296.97, y=62.03, z=-337.19], EntitySheep['Sheep'/23157, l='MpServer', x=-309.69, y=63.00, z=-418.78], EntityItem['item.tile.doublePlant.syringa'/23172, l='MpServer', x=-326.22, y=62.00, z=-344.38], EntityItem['item.tile.gravel'/17045, l='MpServer', x=-328.50, y=20.88, z=-311.47], EntitySheep['Sheep'/17046, l='MpServer', x=-291.50, y=64.00, z=-317.34], EntityItem['item.tile.doublePlant.syringa'/24214, l='MpServer', x=-327.56, y=63.00, z=-348.69], EntitySheep['Sheep'/17050, l='MpServer', x=-307.56, y=63.50, z=-317.19], EntityPig['Pig'/15053, l='MpServer', x=-288.50, y=63.00, z=-284.50], EntityPig['Pig'/15054, l='MpServer', x=-288.50, y=63.00, z=-286.50], EntityPig['Pig'/15055, l='MpServer', x=-288.84, y=63.00, z=-289.44], EntityPig['Pig'/15056, l='MpServer', x=-290.19, y=63.00, z=-295.84], EntityBat['Bat'/30436, l='MpServer', x=-284.50, y=21.09, z=-312.53], EntityBat['Bat'/23436, l='MpServer', x=-337.73, y=21.15, z=-344.13], EntityBat['Bat'/23437, l='MpServer', x=-336.80, y=20.63, z=-351.50], EntitySheep['Sheep'/14222, l='MpServer', x=-335.94, y=67.00, z=-284.69], EntityBat['Bat'/31630, l='MpServer', x=-348.38, y=30.00, z=-389.38], EntitySheep['Sheep'/14223, l='MpServer', x=-337.84, y=67.50, z=-282.44], EntitySheep['Sheep'/14224, l='MpServer', x=-331.22, y=65.00, z=-290.72], EntityBat['Bat'/31632, l='MpServer', x=-342.91, y=30.03, z=-396.41], EntitySheep['Sheep'/14225, l='MpServer', x=-333.50, y=68.50, z=-279.59], EntityBat['Bat'/31633, l='MpServer', x=-344.19, y=30.00, z=-394.50], EntityBat['Bat'/23448, l='MpServer', x=-422.25, y=40.16, z=-298.59], EntityBat['Bat'/31641, l='MpServer', x=-351.63, y=31.09, z=-356.81], EntityBat['Bat'/31644, l='MpServer', x=-319.34, y=45.00, z=-306.47], EntityItem['item.item.bean'/30626, l='MpServer', x=-395.00, y=63.00, z=-351.38], EntityBat['Bat'/23460, l='MpServer', x=-432.03, y=40.00, z=-283.44], EntityBat['Bat'/31655, l='MpServer', x=-313.41, y=31.88, z=-321.50], EntityBat['Bat'/29624, l='MpServer', x=-379.21, y=50.46, z=-318.55], EntityItem['item.tile.doublePlant.syringa'/23486, l='MpServer', x=-325.50, y=63.00, z=-350.00], EntityBat['Bat'/29671, l='MpServer', x=-389.35, y=18.40, z=-331.29]]
    Retry entities: 0 total; []
    Server brand: fml,forge
    Server type: Integrated singleplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:347)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2536)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:370)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.8.9
    Operating System: Windows 10 (amd64) version 10.0
    Java Version: 1.8.0_77, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 2578634144 bytes (2459 MB) / 5205647360 bytes (4964 MB) up to 16965173248 bytes (16179 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx16G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn2G
    IntCache: cache: 0, tcache: 0, allocated: 15, tallocated: 95
    FML: Minecraft 1.8.9 MCP 9.19 Powered by Forge 11.15.1.1855 86 mods loaded, 86 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
    UCHIJAAAA   FML{8.0.99.99} [Forge Mod Loader] (forge-1.8.9-11.15.1.1855.jar) 
    UCHIJAAAA   Forge{11.15.1.1855} [Minecraft Forge] (forge-1.8.9-11.15.1.1855.jar) 
    UCHIJAAAA   OpenComputers|Core{1.6.0-rc.1+07534fd} [OpenComputers (Core)] (minecraft.jar) 
    UCHIJAAAA   securitycraft{v1.8.1} [SecurityCraft] ([1.8.8] SecurityCraft v1.8.1.jar) 
    UCHIJAAAA   ArchitectureCraft{1.4.1} [ArchitectureCraft] (ArchitectureCraft-1.4.1-mc1.8.9.jar) 
    UCHIJAAAA   Circuits{3.0.0} [Automated Redstone] (AutomatedRedstone-1.8.9-3.0.0.jar) 
    UCHIJAAAA   basemetals{1.8.0} [Base Metals] (BaseMetals_1.8.9-1.8.0.jar) 
    UCHIJAAAA   BetterAchievements{0.1.1} [Better Achievements] (BetterAchievements-1.8.9-0.1.1.jar) 
    UCHIJAAAA   bookshelfapi{2.3.7} [Bookshelf] (Bookshelf-API-1.8.9-2.3.7.jar) 
    UCHIJAAAA   BuildCraft|Core{7.2.5} [BuildCraft] (buildcraft-7.2.5.jar) 
    UCHIJAAAA   BuildCraft|Silicon{7.2.5} [BC Silicon] (buildcraft-7.2.5.jar) 
    UCHIJAAAA   BuildCraft|Transport{7.2.5} [BC Transport] (buildcraft-7.2.5.jar) 
    UCHIJAAAA   BuildCraft|Builders{7.2.5} [BC Builders] (buildcraft-7.2.5.jar) 
    UCHIJAAAA   BuildCraft|Energy{7.2.5} [BC Energy] (buildcraft-7.2.5.jar) 
    UCHIJAAAA   BuildCraft|Robotics{7.2.5} [BC Robotics] (buildcraft-7.2.5.jar) 
    UCHIJAAAA   BuildCraft|Factory{7.2.5} [BC Factory] (buildcraft-7.2.5.jar) 
    UCHIJAAAA   BuildCraft|Compat{7.2.1} [BuildCraft Compat] (buildcraft-compat-7.2.1.jar) 
    UCHIJAAAA   buildingbrickscompatvanilla{1.8.9-1.4.4} [Building Bricks Vanilla Compat] (BuildingBricks-1.8.9-1.4.4.jar) 
    UCHIJAAAA   buildingbricks{1.8.9-1.4.4} [Building Bricks] (BuildingBricks-1.8.9-1.4.4.jar) 
    UCHIJAAAA   cashCraft{2.0.0} [CashCraft] (CashCraft-mc1.8.9-2.0.0.jar) 
    UCHIJAAAA   Chameleon{1.8.9-1.1.2} [Chameleon] (Chameleon-1.8.9-1.1.2.jar) 
    UCHIJAAAA   craftingcraft{2.0.13} [CraftingCraft] (craftingcraft-mc1.8.9-2.0.13.jar) 
    UCHIJAAAA   craftingtweaks{3.0.55} [Crafting Tweaks] (craftingtweaks-mc1.8.9-3.0.55.jar) 
    UCHIJAAAA   craftingtweaksaddons{1.0} [Crafting Tweaks Addons] (craftingtweaksaddons-mc1.8.9-3.0.8.jar) 
    UCHIJAAAA   excore{1.4.1} [EXCore] (Excore-1.4.1-1.8.9.jar) 
    UCHIJAAAA   creeperawareness{2.0} [Creeper Awareness] (CreeperAwareness-2.0-1.8.9.jar) 
    UCHIJAAAA   customnpcs{1.8.9_beta} [CustomNPCs] (CustomNpcs_1.8.9_beta(21feb16).jar) 
    UCHIJAAAA   decorationmegapack{1.8.9.2} [Decoration Mega Pack] (decorationmegapack-1.8.9.2.jar) 
    UCHIJAAAA   dlr{2.4.1} [Don's Lightning Rod] (Don's Lightning Rod-1.8.9-2.4.1.jar) 
    UCHIJAAAA   dsurround{2.0.5.6} [Dynamic Surroundings] (DynamicSurroundings-1.8.9-2.0.5.6.jar) 
    UCHIJAAAA   ea{v1.4.1a-mc1.8.9} [Ebon Arts] (EbonArts-v1.4.1a-mc1.8.9.jar) 
    UCHIJAAAA   poweradvantage{1.5.3} [Power Advantage] (PowerAdvantage-1.5.3.jar) 
    UCHIJAAAA   electricadvantage{1.1.2} [Electric Advantage] (ElectricAdvantage-1.1.2.jar) 
    UCHIJAAAA   ShetiPhianCore{3.1.0} [ShetiPhian-Core] (ShetiPhianCore-1.8.9-3.1.0.jar) 
    UCHIJAAAA   EnderTanks{1.2.1} [EnderTanks] (EnderTanks-1.8.9-1.2.1.jar) 
    UCHIJAAAA   enderutilities{0.5.3} [Ender Utilities] (enderutilities-1.8.9-0.5.3.jar) 
    UCHIJAAAA   golems{3.04} [Extra Golems] (Extra-Golems[1.8.9]-3.04.jar) 
    UCHIJAAAA   Waila{1.6.0} [Waila] (Waila-1.6.0-B3_1.8.8.jar) 
    UCHIJAAAA   golems_addon_waila{3.02} [Extra Golems Waila Addon] (ExtraGolemsWailaAddon[1.8.9]-3.02.jar) 
    UCHIJAAAA   fe{1.1} [Food Expansion] (FoodExpansion1.1-mc1.8.9.jar) 
    UCHIJAAAA   gemsplus{1.6.4} [Gems+] (gemsplus-1.6.4-mc1.8.8.jar) 
    UCHIJAAAA   hopperducts{1.4.6} [Hopper Ducts] (hopperducts-mc1.8.8-1.4.6.jar) 
    UCHIJAAAA   Automation{4.0.2} [Inductive Automation] (InductiveAutomation-1.8.9-4.0.3.jar) 
    UCHIJAAAA   IC2{2.3.225-ex18} [IndustrialCraft 2] (industrialcraft-2-2.3.225-ex18.jar) 
    UCHIJAAAA   infiniteinvo{1.1.51} [InfiniteInvo] (InfiniteInvo-1.1.51.jar) 
    UCHIJAAAA   LunatriusCore{1.1.2.32} [LunatriusCore] (LunatriusCore-1.8.9-1.1.2.32-universal.jar) 
    UCHIJAAAA   InGameInfoXML{2.8.1.80} [InGame Info XML] (InGameInfoXML-1.8.9-2.8.1.80-universal.jar) 
    UCHIJAAAA   JEI{2.28.16.184} [Just Enough Items] (jei_1.8.9-2.28.16.184.jar) 
    UCHIJAAAA   journeymap{1.8.9-5.1.5} [JourneyMap] (journeymap-1.8.9-5.1.5-unlimited.jar) 
    UCHIJAAAA   jeresources{0.4.0} [Just Enough Resources] (JustEnoughResources-1.8.9-0.4.0.jar) 
    UCHIJAAAA   malisiscore{1.8.9-2.1.3} [MalisisCore] (malisiscore-1.8.9-2.1.3.jar) 
    UCHIJAAAA   malisisadvert{1.8.9-2.0.1} [Malisis Advert] (malisisadvert-1.8.9-2.0.1.jar) 
    UCHIJAAAA   malisisblocks{1.8-2.0.0} [Malisis Blocks] (malisisblocks-1.8-2.0.0.jar) 
    UCHIJAAAA   malisisdoors{1.8.9-3.1.3} [MalisisDoors] (malisisdoors-1.8.9-3.1.3.jar) 
    UCHIJAAAA   malisisswitches{1.8.9-1.0.1} [Malisis Switches] (malisisswitches-1.8.9-1.0.1.jar) 
    UCHIJAAAA   mantle{1.8.9-0.9.2.jenkins102} [Mantle] (Mantle-1.8.9-0.9.2.jar) 
    UCHIJAAAA   mcmultipart{1.0.9} [MCMultiPart] (MCMultiPart-1.0.9-universal.jar) 
    UCHIJAAAA   moarsigns{2.1.0} [MoarSigns] (MoarSigns-1.8.9-2.1.0.jar) 
    UCHIJAAAA   tconstruct{1.8.9-2.2.1.jenkins1005} [Tinkers' Construct] (TConstruct-1.8.9-2.2.1.jar) 
    UCHIJAAAA   neotech{2.4.4} [NeoTech] (NeoTech-1.8.9-2.4.4.jar) 
    UCHIJAAAA   potioncore{1.0.1_for_1.8.9} [Potion Core] (PotionCore-1.0.1_for_1.8.9.jar) 
    UCHIJAAAA   nep{1.2_for_1.8.9} [Not Enough Potions] (NotEnoughPotions-1.2_for_1.8.9.jar) 
    UCHIJAAAA   offlineskins{1.8.9-v3c} [OfflineSkins] (OfflineSkins-1.8.9-v3c.jar) 
    UCHIJAAAA   OpenComputers{1.6.0-rc.1+07534fd} [OpenComputers] (OpenComputers-MC1.8.9-1.6.0-rc.1+07534fd.jar) 
    UCHIJAAAA   openradio{0.7.0} [OpenRadio] (OpenRadio-0.7.0-beta-MC1.8.9.jar) 
    UCHIJAAAA   fodc{1.7.2} [Ore Dictionary Converter] (OreDictionaryConverter-1.7.2.jar) 
    UCHIJAAAA   plantmegapack{1.8.9.21} [Plant Mega Pack] (plantmegapack-1.8.9.21.jar) 
    UCHIJAAAA   ProjectE{1.8.9-PE1.1.4} [ProjectE] (ProjectE-1.8.9-PE1.1.4.jar) 
    UCHIJAAAA   randomthings{3.6.6} [Random Things] (RandomThings-MC1.8.9-3.6.6.jar) 
    UCHIJAAAA   redstonepaste{1.7.2} [Redstone Paste] (redstonepaste-mc1.8.8-1.7.2.jar) 
    UCHIJAAAA   Roguelike{1.4.5} [Roguelike Dungeons] (RoguelikeDungeons-1.8.9-1.4.5.jar) 
    UCHIJAAAA   sciapi{1.2.0.1} [SciAPI] (SciAPI-1.2.0.1.jar) 
    UCHIJAAAA   sensorcraft{0.1.1} [SensorCraft] (sensorcraft-0.1.1-MC1.8.9.jar) 
    UCHIJAAAA   steamadvantage{1.5.3} [Steam Advantage] (SteamAdvantage-1.5.3.jar) 
    UCHIJAAAA   stellarsky{0.2.2.7b} [Stellar Sky] (Stellar Sky-0.2.2.7b.jar) 
    UCHIJAAAA   StorageDrawers{1.8.9-2.6.4} [Storage Drawers] (StorageDrawers-1.8.9-2.6.4.jar) 
    UCHIJAAAA   storagesilo{1.2.2} [StorageSilo] (storagesilo-1.8.9-1.2.2b.jar) 
    UCHIJAAAA   terrariabuttons{1.0.1} [Terraria Buttons] (TerrariaButtons-1.8.8-1.0.1.jar) 
    UCHIJAAAA   thutcore{2.4.6} [ThutCore] (thutcore-1.8.9-2.4.6.jar) 
    UCHIJAAAA   thuttech{2.3.7} [Thut's Tech] (thuttech-2.3.7.jar) 
    UCHIJAAAA   tis3d{0.8.4.15} [TIS-3D] (TIS-3D-MC1.8.9-0.8.4.15.jar) 
    UCHIJAAAA   unlogicii{4.2.1.51} [UnLogic II] (UnLogicII-4.2.1.51.jar) 
    UCHIJAAAA   unlogiciivc{1.5} [UnLogic II Version Checker] (UnLogicII-4.2.1.51.jar) 
    UCHIJAAAA   worldedit{6.1.1} [WorldEdit] (worldedit-forge-mc1.8.9-6.1.1.jar) 
    UCHIJAAAA   WorldEditCuiFe{1.0.2} [WorldEdit CUI Forge Edition] (WorldEditCuiFe-v1.0.2-mf-1.8.9-11.15.1.1764.jar) 
    Loaded coremods (and transformers): 
NowWithRendering (hopperducts-mc1.8.8-1.4.6.jar)
  net.fybertech.nwr.NowWithRendering
TransformerLoader (OpenComputers-MC1.8.9-1.6.0-rc.1+07534fd.jar)
  li.cil.oc.common.asm.ClassTransformer
MalisisSwitchesPlugin (malisisswitches-1.8.9-1.0.1.jar)
  net.malisis.switches.asm.MalisisSwitchesTransformer
MalisisCorePlugin (malisiscore-1.8.9-2.1.3.jar)
  net.malisis.core.util.chunkcollision.ChunkCollisionTransformer
  net.malisis.core.util.chunkblock.ChunkBlockTransformer
  net.malisis.core.renderer.transformer.MalisisRendererTransformer
  net.malisis.core.renderer.icon.asm.TextureMapTransformer
IC2core (industrialcraft-2-2.3.225-ex18.jar)

NowWithRendering (redstonepaste-mc1.8.8-1.7.2.jar)
  net.fybertech.nwr.NowWithRendering
dsurroundcore (DynamicSurroundings-1.8.9-2.0.5.6.jar)
  org.blockartistry.mod.DynSurround.asm.Transformer
offlineskins (OfflineSkins-1.8.9-v3c.jar)
  lain.mods.skins.asm.ASMTransformer
LoadingPlugin (RandomThings-MC1.8.9-3.6.6.jar)
  lumien.randomthings.asm.ClassTransformer
    GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 353.54' Renderer: 'GeForce GTX 980M/PCIe/SSE2'
    Pulsar/tconstruct loaded Pulses: 
        - TinkerCommons (Enabled/Forced)
        - TinkerWorld (Enabled/Not Forced)
        - TinkerTools (Enabled/Not Forced)
        - TinkerSmeltery (Enabled/Not Forced)
        - TinkerGadgets (Enabled/Not Forced)
        - TinkerOredict (Enabled/Forced)
        - TinkerIntegration (Enabled/Forced)
        - TinkerFluids (Enabled/Forced)
        - TinkerMaterials (Enabled/Forced)
        - craftingtweaksIntegration (Enabled/Not Forced)

    Launched Version: 1.8.9-forge1.8.9-11.15.1.1855
    XRay Version: 15
    LWJGL: 2.9.4
    OpenGL: GeForce GTX 980M/PCIe/SSE2 GL version 4.5.0 NVIDIA 353.54, NVIDIA Corporation
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

    Using VBOs: No
    Is Modded: Definitely; Client brand changed to 'fml,forge'
    Type: Client (map_client.txt)
    Resource Packs: 
    Current Language: English (US)
    Profiler Position: N/A (disabled)
    CPU: 8x Intel(R) Core(TM) i7-4710HQ CPU @ 2.50GHz
```

This is using my custom build I mentioned in another bug report here.

Removing OC (or IC2) fixes it, though I would like to use both.

**Comment by LizzyTrickster** [comment]
Have you tried with just IC2 and OC? Seems you have a few other mods in there as well

**Comment by fnuecke** [comment]
I've seen that scrolling by, but it never crashed me. I assumed it was a JEI vs. IC2 issue because it happened when opening the inventory, and OC isn't mentioned in the trace. I'll look into it.

**Comment by brisingraerowing** [comment]
I could open the inventory, I just couldn't select the Search Items tab. The IC2 tabs were kind of hit or miss on whether they would crash. No other tabs would crash. Just IC2.

And @LizzyTrickster I did try with just IC2 and OC, and got a garbled crash log (codepage error I think. Happens quite a bit I noticed), although the cause was the same, so I just posted this one.

@fnuecke I tried without JEI first and it still crashed.
